### PR TITLE
Forms Automated Test - LPS-131224 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**

### CODE REVIEW

**Comment by brittneyq** [comment]
@natocesarrego @diogocordeiro this should be ready for review!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 1]
```diff
@@ -51,12 +51,50 @@ definition {
...
+		AssertElementPresent(
+			key_bodyText = "Test",
+			key_fieldName = "Paragraph",
+			locator1 = "FormFields#FIELD_TARGET_LABEL_AND_BODY_TEXT");
+
+		AssertElementPresent(
+			key_bodyText = "Test",
+			key_fieldName = "Copy of Paragraph",
+			locator1 = "FormFields#FIELD_TARGET_LABEL_AND_BODY_TEXT");
 	}
```
Comment (diogocordeiro) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:98]: Hi @LucasAcacio, 

I am not sure but I think this last step on the test is missing --> 
10. Assert the new Paragraph field's field reference is different from the field reference of the original Paragraph field. 

Can you verify this, please? 

Thanks!

**Review by diogocordeiro — CHANGES_REQUESTED** [review]

**Comment by brittneyq** [comment]
Hi @LucasAcacio, there seems to be conflicting files with this PR for `portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path`
 would you mind making the appropriate changes?

**Comment by diogocordeiro** [comment]
ci:forward

**Pull request closed without merging**